src/_pytest/terminal.py

```diff
@@ -426,6 +427,28 @@ def ensure_newline(self) -> None:
             self._tw.line()
             self.currentfspath = None
 
+    def wrap_write(
```

Comment:
Not sure "wrap" is appropriated, given "wrap" in this contents is to preserve the text, but wrapping it around at certain width (for example 70). I think what we are doing here is actually *truncating*, so perhaps `truncate_write` or something would be more appropriate?

Reply:
Happy to rename to whatever, but in this case, the new method is only wrapping text.  The truncated output you saw was unmodified behavior due to the test testing multiple outputs.  
I'm not particularly satisfied with the name because I'm not sure that it feels sufficiently conventional.  Perhaps `write_wrapped_text` or something similar would be better?

Reply:
Indeed, I misunderstood the output hence my misguided suggestion. `wrap_write` is fine, thanks! 👍